tests/PHPUnit/Integration/Tracker/RequestSetTest.php

```diff
@@ -128,49 +128,6 @@ private function buildRequest($idsite)
         return $request;
     }
 
-    public function test_shouldPerformRedirectToUrl_shouldNotRedirect_IfNoUrlIsSet()
```

Comment:
I guess the class `TestRequestSet` in the beginning of the file can now be adjusted and the redirect stuff can be removed from it.

Reply:
fixed now as well